src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,43 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  health = 100;
+
+  static alive = [];
+  static addToAlive(obj) {
+    this.alive.push(obj);
+  }
+
+  static deleteDead() {
+    const index = this.alive.findIndex(animal => animal.health === 0);
+
+    this.alive.splice(index, 1);
+  }
+
+  constructor(name) {
+    this.name = name;
+
+    Animal.addToAlive(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+  hide() {
+    this.hidden = true;
+  }
```

Comment:
Method hide should change hidden property to the opposite value, not always to the true